importer/parsexml.py

```diff
@@ -180,6 +181,9 @@ def parse_element_tree(tree, filename=None):
         if all(sections):
             article_data["country"] = sections[0]
 
+    for heading in root.iter("MeshHeadingList"):
+        article_data["Topic"] = heading.find('DescriptorName').text
```

Comment:
Does every article have this field? You can use `parse_selected_sections` to handle the case where it is `None`